scripts/build.js

```diff
@@ -14,6 +14,11 @@ process.on('unhandledRejection', err => {
 // Ensure environment variables are read.
 require('../config/env');
 
+// Default to ropsten test network
+process.env.TARGET_NETWORK = process.env.TARGET_NETWORK || 'ropsten';
+process.env.DEFAULT_GAS = process.env.DEFAULT_GAS || 6721975;
+process.env.DEFAULT_GAS_PRICE = process.env.DEFAULT_GAS_PRICE || 20000000000;
```

Comment:
Might be worth adding instructions to readme on how to get ropsten eth.

Reply:
Good point, I've updated the readme a little bit.

Reply:
Note that this is also already covered in the developer's handbook: https://github.com/magmo/developer-handbook/blob/master/deploying-contracts-with-truffle.md